<?xml version="1.0" encoding="UTF-8"?><testrun name="DoubleVectorTester" project="215HWA2" tests="17" started="17" failures="1" errors="0" ignored="0">
  <testsuite name="DoubleVectorTester" time="0.805">
    <testcase name="testLotsOfDenseSets" classname="DoubleVectorTester" time="0.033"/>
    <testcase name="testL1NormSparseShort" classname="DoubleVectorTester" time="0.008"/>
    <testcase name="testLotsOfAdds" classname="DoubleVectorTester" time="0.276">
      <failure>junit.framework.AssertionFailedError: Got 0.0 at pos 0, expected 11607.750000000002
	at junit.framework.Assert.fail(Assert.java:57)
	at junit.framework.TestCase.fail(TestCase.java:227)
	at DoubleVectorTester.checkCloseEnough(DoubleVectorTester.java:24)
	at DoubleVectorTester.testLotsOfAdds(DoubleVectorTester.java:428)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at junit.framework.TestCase.runTest(TestCase.java:176)
	at junit.framework.TestCase.runBare(TestCase.java:141)
	at junit.framework.TestResult$1.protect(TestResult.java:122)
	at junit.framework.TestResult.runProtected(TestResult.java:142)
	at junit.framework.TestResult.run(TestResult.java:125)
	at junit.framework.TestCase.run(TestCase.java:129)
	at junit.framework.TestSuite.runTest(TestSuite.java:255)
	at junit.framework.TestSuite.run(TestSuite.java:250)
	at org.junit.internal.runners.JUnit38ClassRunner.run(JUnit38ClassRunner.java:84)
	at org.eclipse.jdt.internal.junit4.runner.JUnit4TestReference.run(JUnit4TestReference.java:50)
	at org.eclipse.jdt.internal.junit.runner.TestExecution.run(TestExecution.java:38)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:459)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.runTests(RemoteTestRunner.java:675)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.run(RemoteTestRunner.java:382)
	at org.eclipse.jdt.internal.junit.runner.RemoteTestRunner.main(RemoteTestRunner.java:192)

</failure>
    </testcase>
    <testcase name="testNormalizeDense" classname="DoubleVectorTester" time="0.044"/>
    <testcase name="testL1NormDenseShort" classname="DoubleVectorTester" time="0.002"/>
    <testcase name="testOutOfBounds" classname="DoubleVectorTester" time="0.005"/>
    <testcase name="testLotsOfSparseSets" classname="DoubleVectorTester" time="0.154"/>
    <testcase name="testLotsOfSparseSetsWithAddToAll" classname="DoubleVectorTester" time="0.067"/>
    <testcase name="testL1NormSparseLong" classname="DoubleVectorTester" time="0.034"/>
    <testcase name="testSingleSparseSetSize1" classname="DoubleVectorTester" time="0.009"/>
    <testcase name="testSingleDenseSetSize1" classname="DoubleVectorTester" time="0.006"/>
    <testcase name="testSingleSparseSetSize1000" classname="DoubleVectorTester" time="0.003"/>
    <testcase name="testRounding" classname="DoubleVectorTester" time="0.052"/>
    <testcase name="testNormalizeSparse" classname="DoubleVectorTester" time="0.074"/>
    <testcase name="testSingleDenseSetSize1000" classname="DoubleVectorTester" time="0.001"/>
    <testcase name="testL1NormDenseLong" classname="DoubleVectorTester" time="0.004"/>
    <testcase name="testLotsOfDenseSetsWithAddToAll" classname="DoubleVectorTester" time="0.022"/>
  </testsuite>
</testrun>
